src/_base/harness/scripts/destroy.sh

```diff
@@ -9,6 +9,7 @@ if [[ "$USE_DOCKER_SYNC" = "yes" ]]; then
 fi
 
 if [[ "$APP_BUILD" = "static" ]]; then
+    # shellcheck disable=SC2046
```

Comment:
As we are intentionally splitting the word here, I'm not sure if we can fix this shellcheck error.